### PR TITLE
Add PaddleEasyDL 1.1.3

### PR DESCRIPTION
**Pull request by PeterPanZH**
**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

_In the following questions `<cask>` is the token of the cask you're submitting._

After making all changes to a cask, verify:

- [x] The submission is for [a stable version](https://docs.brew.sh/Acceptable-Casks#stable-versions) or [documented exception](https://docs.brew.sh/Acceptable-Casks#but-there-is-no-stable-version).
- [x] `brew audit --cask <cask>` is error-free.
- [x] `brew style --fix <cask>` reports no offenses.

Additionally, **if adding a new cask**:

- [x] Named the cask according to the [token reference](https://docs.brew.sh/Cask-Cookbook#token-reference).
- [x] Checked the cask was not [already refused](https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues).
- [x] Checked the cask is submitted to [the correct repo](https://docs.brew.sh/Acceptable-Casks#finding-a-home-for-your-cask).
- [x] `brew audit --new-cask <cask>` worked successfully.
- [x] `brew install --cask <cask>` worked successfully.
- [x] `brew uninstall --cask <cask>` worked successfully.

### CODE REVIEW

File: Casks/paddle-easydl.rb [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+      match = page.match(%r{path:.*/飞桨EasyDL-(\d+).(\d+).(\d+).(\d+).zip}i)
+      next if match.blank?
+
+      "#{match[1]}.#{match[2]}.#{match[3]},#{match[4]}"
+    end
+  end
+
+  auto_updates true
+  depends_on macos: ">= :el_capitan"
+  depends_on arch: :intel
```
Comment (neutric) [inline Casks/paddle-easydl.rb:23]: Why does this depend on `arch: :intel`? Wouldn't Rosetta 2 work?

Comment (PeterPanZH) [reply Casks/paddle-easydl.rb:23]: This is a deep learning development tool that requires x86_64 arch. Some feature may not work properly under Rosetta 2 on Apple Silicon.

Comment (SMillerDev) [reply Casks/paddle-easydl.rb:23]: Otherwise it would need the architecture to be "intel"
```suggestion
  depends_on arch: :x86_64
```

Comment (PeterPanZH) [reply Casks/paddle-easydl.rb]: Fixed. Though according to the [official cookbook](https://docs.brew.sh/Cask-Cookbook#depends_on-arch) these two symbols have same meaning... for now at least.

Comment (neutric) [reply Casks/paddle-easydl.rb]: 👍 The change is good nonetheless as all other Casks needing this use `:x86_64`.

**Pull request merged**